amethyst_renderer/src/mouse.rs

```diff
@@ -11,7 +11,7 @@ pub fn hide_cursor(msg: &mut WindowMessages) {
 /// Set the cursor back to normal/visible.
 pub fn release_cursor(msg: &mut WindowMessages) {
     msg.send_command(move |win| {
-        if let Err(err) = win.grab_cursor(false) {
```

Comment:
You'll notice this passes in `false` and the grab_cursor function passes in `true.